models/src/test/scala/coop/rchain/models/RhoTypesTest.scala

```diff
@@ -0,0 +1,39 @@
+package coop.rchain.models
+
+import org.scalacheck.Gen
+import org.scalatest.{Matchers, PropSpec}
+import org.scalatest.prop.PropertyChecks
+
+class RhoTypesTest extends PropSpec with PropertyChecks with Matchers {
+  def rtt[T](msg: T, serializeInstance: Serialize[T]): Boolean = {
+    val bytes = serializeInstance.encode(msg)
+    val msg2  = serializeInstance.decode(bytes)
+    assert(msg2.isRight && msg == msg2.right.get)
+
+    val errBytes = Array[Byte](1, 2, 3, 4)
+    val msg3     = serializeInstance.decode(errBytes)
+    assert(msg3.isLeft && msg3.left.get.isInstanceOf[SerializeError])
+    true
```

Comment:
you can remove this and have the return type of `rtt` be `Unit`

Reply:
Done.